maro/rl/agent/abs_agent.py

```diff
@@ -22,17 +28,16 @@ class AbsAgent(ABC):
 
     Args:
         name (str): Agent's name.
+        mode (AgentMode): An ``AgentMode`` enum member that indicates the role of the agent in the current process.
         algorithm (AbsAlgorithm): A concrete algorithm instance that inherits from AbstractAlgorithm.
             This is the centerpiece of the Agent class and is responsible for the most important tasks of an agent:
             choosing actions and optimizing models.
-        experience_pool (AbsStore): A data store that stores experiences generated by the experience shaper.
+        experience_pool (AbsStore): A data store that stores experiences generated by the experience shaper. This is
+            only necessary for some algorithms. Defaults to None.
```

Comment:
It is used to store experiences processed by the experience shaper, which will be used by some value-based algorithms, such as DQN. Default is None.

Reply:
"Defaults to None" per Jinyu